ui/components/component-library/avatar-base/avatar-base.tsx

```diff
@@ -47,14 +52,14 @@ export const AvatarBase = forwardRef(
           className,
         )}
         ref={ref}
-        as={ValidTag.Div}
+        as="div"
         display={Display.Flex}
         justifyContent={JustifyContent.center}
         alignItems={AlignItems.center}
         borderRadius={BorderRadius.full}
         variant={fallbackTextVariant}
         textTransform={TextTransform.Uppercase}
-        {...{ backgroundColor, borderColor, color, ...props }}
+        {...{ backgroundColor, borderColor, color, ...props }} // TODO: Fix type error to do with spreading props in Box types
       >
         {children}
       </Text>
```

Comment:
Check if the updates to this file need to be made it would be better if they did not.

Reply:
Removed.

Reply:
Turns out we do need these because we are updating the `TextProps`